index.bs

```diff
@@ -474,8 +474,17 @@ An implementation may define <dfn export>extension modules</dfn>. These must hav
 [=module name=] that contains a single colon "<code>:</code>" character. The
 part before the colon is the prefix; this is typically the same for all
 [=extension modules=] specific to a given implementation and should be unique for a
-given implementation. Such modules extend the [=local end definition=] and [=remote end definition=]
-providing additional groups as choices for the defined [=commands=] and [=events=].
+given implementation.
+Such modules extend the [=local end definition=] and [=remote end definition=]
+providing additional groups as choices for the defined [=commands=], [=events=] and types.
+
+<dfn export>External specifications</dfn> may define their own modules that
```

Comment:
The only purpose of this `dfn` is to have a place to link to from external specifications. Open to other suggestions.

Reply:
I think the definition is not needed

Reply:
How do we link to this part from other PRs then?

Reply:
I don't think we need to, we just link to the whole spec.

Reply:
OK, removed.